app/components/work_history_review_component.html.erb

```diff
@@ -1,15 +1,19 @@
-<% @application_form.application_work_experiences.order(:start_date).each do |work| %>
-  <%= render(SummaryCardComponent, rows: work_experience_rows(work), editable: @editable) do %>
-    <%= render(SummaryCardHeaderComponent, title: work.role, heading_level: @heading_level, check_icon: work.working_with_children) do %>
+<%@application_form.application_work_experiences.order(:start_date).to_a.append(nil).each_cons(2) do |work1, work2| %>
```

Comment:
Pseudocode for how component could deal with the situation where one of the first work experiences is a current work (it goes into the present, has a nil `end_date`)

```ruby
<% work_experiences = @application_form.application_work_experiences.order(:start_date) %>
<% work_experiences_with_breaks, work_experiences_without_breaks = work_experiences.to_a.partition { |w| w.end_date.nil? }

<% work_experiences_with_breaks.append(work_experiences_without_breaks.first).each_cons(2) do |work1, work2| %>
  <%= render(IndividualWorkExperienceReviewComponent, ...) %>
  <%= render(BreakInWorkHistoryComponent, work1: work1, work2: work2) %>
<% end %>

<% work_experiences_without_breaks.each do |work| %>
  <%= render(IndividualWorkExperienceReviewComponent, ...) %>
<% end %>
```

This can be done as a follow-up PR!